templates/argo-tasks/generate-path.yml

```
@@ -29,5 +30,14 @@ spec:
           - name: AWS_ROLE_CONFIG_PATH
             value: s3://linz-bucket-config/config.json
         args:
-          - 'lint-inputs'
-          - '{{inputs.parameters.path}}'
+          - 'generate-path'
+          - '--target-bucket-name'
+          - '{{inputs.parameters.target_bucket_name}}'
```

It might be clearer to pair up the key/value pair:

```suggestion
          - '--target-bucket-name={{inputs.parameters.target_bucket_name}}'
```

---

I agree that matching the values would make it clearer here, however I am following the structure that is used across topo-workflows. It is ok to just change this instance or should this be a change made consistently across the repo?

---

Feel free to skip it now; we could do it as a separate PR.